internal/report.go

```diff
@@ -539,3 +539,32 @@ func writeHeading(w *bufio.Writer, s string) {
 		s, "\n",
 		"----------------------------\n"}, ""))
 }
+
+// Provides a description and severity for each schema issue.
+// Note on batch: for some issues, we'd like to report just the first instance
+// in a table and suppress other instances i.e. adding more instances
+// of the issue in the same table has little value and could be very noisy.
+// This is controlled via 'batch': if true, we count only the first instance
+// for assessing warnings, and we give only the first instance in the report.
+// TODO: add links in these descriptions to further documentation
+// e.g. for timestamp description.
+var IssueDB = map[SchemaIssue]struct {
```

Comment:
Isn't this a copy of issueDB defined earlier in this file?
Shouldn't we just change brief to Brief and issueDB to IssueDB and avoid copying all of this?

Reply:
yes, updated IssueDB